.github/workflows/fedora-rawhide.yml

```diff
@@ -26,15 +26,11 @@ jobs:
       run: |
         ./autogen.sh
         ./configure
```

Comment:
The `./configure` step is not needed for cmake builds.

(I also think `./autogen.sh` isn't really needed and `./update.sh` would be good enough, but let's leave it like that for now).